internal/service/alertconfiguration/data_source_alert_configuration_test.go

```diff
@@ -237,3 +238,11 @@ func configWithPagerDutyDS(projectID, serviceKey string, enabled bool) string {
 		}
 	`, projectID, serviceKey, enabled)
 }
+
+func checkExists(resourceName string) resource.TestCheckFunc {
```

Comment:
is checkExists still being used? maybe in mig tests?

Reply:
correct, left to not make additional changes on that side